celery/bin/celery.py

```diff
@@ -160,7 +130,26 @@ def celery(ctx, app, broker, result_backend, loader, config, workdir,
         os.environ['CELERY_CONFIG_MODULE'] = config
     if skip_checks:
         os.environ['CELERY_SKIP_CHECKS'] = 'true'
-    ctx.obj = CLIContext(app=app, no_color=no_color, workdir=workdir,
+
+    try:
+        app_object = find_app(app)
```

Comment:
This line crashes in our celery application from `5.0.0rc2`. As our application worked with `rc1` I'm wondering if this is a bug (i.e. worth opening an issue) or some mistake in our startup procedure. What do you think @Nusnus? 

```
celery_app = Celery("modulename") 
...
celery_app.worker_main(argv)
# calls celery_app.start() > celery.main() > celery(...) > find_app(app)

Traceback (most recent call last):
  File ".../lib/python3.12/site-packages/celery/bin/celery.py", line 135, in celery
    app_object = find_app(app)
                 ^^^^^^^^^^^^^
  File ".../lib/python3.12/site-packages/celery/app/utils.py", line 383, in find_app
    sym = symbol_by_name(app, imp=imp)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.12/site-packages/kombu/utils/imports.py", line 59, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.12/site-packages/celery/utils/imports.py", line 109, in import_from_cwd
    return imp(module, package=package)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.7_1/Frameworks/Python.framework/Versions/3.12/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named '<Celery modulename at 0x...>'
```

Reply:
You're absolutely right, @david-elias-kuenstle-webcom! My bad, sorry. 🙈 

* https://github.com/celery/celery/pull/9426